pkg/helpers/scheme/scheme.go

```diff
@@ -31,7 +29,6 @@ func InstallSchemes(scheme *apimachineryruntime.Scheme) {
 	utilruntime.Must(InstallNonCRDSecurity(scheme))
 	utilruntime.Must(template.Install(scheme))
 	utilruntime.Must(user.Install(scheme))
-	legacy.InstallExternalLegacyAll(scheme)
```

Comment:
You can remove all codes (basically everything under `legacy` folder) in a new commit as they will not be used anymore and finally we get rid of them.

Reply:
`legacy.InstallExternalLegacyAll` is still used in https://github.com/openshift/oc/blob/master/pkg/helpers/graph/genericgraph/test/runtimeobject_nodebuilder.go?plain=1#L136.
 I see that's a test helper of some sort. Do you think it's safe to remove the call from there as well?

Reply:
I gave a stab at removing the call from the graph builder and got some unit test failures. Having a look so that I can estimate whether it's best to do this here or as a follow-up PR.

Reply:
this change does indeed introduce a lot more changes... there are several references to the `legacy` package in `pkg/helpers`. I think it's best to open a separate PR. WDYT?

Reply:
I'm shockingly surprised to see the usage in https://github.com/openshift/oc/blob/master/pkg/helpers/graph/genericgraph/test/runtimeobject_nodebuilder.go?plain=1#L136. You are right, it is better to spend time on this in a follow up not in here.